config/config/wc-config.yaml

```
@@ -128,6 +128,14 @@ hnc:
   ## Extra excluded namespace, here we should add the namespaces that are more likely to change
   excludedExtraNamespaces: []
 
+  ## Additional resources to enable opt-in propagation for.
+  ## Objects that should be propagated must have one of the annotations listed here https://github.com/kubernetes-sigs/hierarchical-namespaces/blob/master/docs/user-guide/how-to.md#limit-the-propagation-of-an-object-to-descendant-namespaces
+  additionalPropagatedResources: []
```

Thanks this works really well, but I think the key name could be a bit clearer to what it actually does e.g.
```suggestion
  additionalAllowPropagateResources: []
```